src/marqo/tensor_search/enums.py

```diff
@@ -80,6 +80,22 @@ class EnvVars:
     MARQO_MAX_DOC_BYTES = "MARQO_MAX_DOC_BYTES"
     MARQO_MAX_RETRIEVABLE_DOCS = "MARQO_MAX_RETRIEVABLE_DOCS"
     MARQO_MODELS_TO_PRELOAD = "MARQO_MODELS_TO_PRELOAD"
+    MARQO_MAX_CONCURRENT_INDEX = "MARQO_MAX_CONCURRENT_INDEX"
+    MARQO_MAX_CONCURRENT_SEARCH = "MARQO_MAX_CONCURRENT_SEARCH"
+    MARQO_THREAD_EXPIRY_TIME = "MARQO_THREAD_EXPIRY_TIME"
+    MARQO_ENABLE_THROTTLING = "MARQO_ENABLE_THROTTLING"
+
+
+class RequestType:
+    INDEX = "INDEX"
+    SEARCH = "SEARCH"
+    DELETE = "DELETE"
+    CREATE = "CREATE"
+
+
+class ThrottleType:
+    thread_count = "thread_count"
+    data_size = "data_size"
```

Comment:
I think I forgot to use it, but ideally we set it up so that when other throttling types are implemented, we start using this. should it be removed?

Reply:
at the moment, no, but it was put in for future throttling methods to be added

Reply:
We can keep it this time, but I'd prefer not to preemptively add these sorts of vars because priorities and architectures change dramatically before they get a chance to be implemented.

I'd prefer to have a comment like this: `# Perhaps create a ThrottleType to differentiate thread_count and data_size throttling mechanisms`

Reply:
added